.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,6 @@
+{
+  "ExpandedNodes": [
+    ""
+  ],
```

Comment:
There are some small files that got changed when Joe was pushing his changes.  Not sure why this happened, but it does not effect anything.